templates/base.html

```diff
@@ -36,6 +36,8 @@
               <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Shop <span class="caret"></span></a>
               <ul class="dropdown-menu" role="menu">
                 <li><a href="#">Apps</a></li>
+                <li><a href="{% url "multimedia:application_list" %}">Apps</a></li>
+                <li><a href="#">Music</a></li>
                 <li><a href="{% url "multimedia:music_list" %}">Music</a></li>
```

Comment:
You should replace the # with the url instead of adding another li. This is the reason why you have duplicate Apps and Music